must-gather/collection-scripts/gather_apiservices

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+
+# Resource list
+resources=()
+
+for i in $(/usr/bin/oc get apiservices --all-namespaces | grep kata | awk '{ print $1 }')
+do
+  resources+=("$i")
+done
+
+# we use nested loops to nicely output objects partitioned per namespace, kind
```

Comment:
Does this comment still apply?

Reply:
Regardless, this reasoning behind this PR is to move the content of whatever is in my personal repo to this one, as the content is, to avoid losing history.

I'd say any improvement would be super welcome and can come as a pull-request **after** this series gets merged, as the PRs were already reviewed, verified by Red Hat QE, and merged in the previous repo.

Does this sound reasonable, @pmores ?

Reply:
Fair enough!